tests/__init__.py

```diff
@@ -0,0 +1,28 @@
+import os
+import shutil
+
+SUPERSET_TEST_HOME = os.path.join(os.path.dirname(__file__), '.superset')
+shutil.rmtree(SUPERSET_TEST_HOME, ignore_errors=True)
+os.environ['SUPERSET_HOME'] = SUPERSET_TEST_HOME
+
+def init_superset_for_testing():
+
+	"""Basic initialization for all tests"""
+
+	# DRAGONS: don't import until ya got the home directory sorted
+	import superset
```

Comment:
I could dig into superset to see if it has some way to initialize more configurably - but basically we gotta do the import here since the import triggers the initialization.